data/enemyBlueprintTables.json

```diff
@@ -1 +1 @@
acee5e140474aa7be2d48851a6c65e","enemyName":"Hunhow","enemyItemDropChance":"50.00","items":[{"_id":"4d0e2ed574d54e9241af02f265181b2b","itemName":"Scimitar Engines Blueprint","rarity":"Rare","chance":2.01},{"_id":"eb7d91507b0d34151f911dad2b6b249a","itemName":"Dread Blueprint","rarity":"Common","chance":75.88},{"_id":"dd1fb9991ea7536bcca2a3d961c5f7aa","itemName":"Despair Blueprint","rarity":"Uncommon","chance":11.06},{"_id":"efdd455f02c082c6e46b6d3c70c89acb","itemName":"Hate Blueprint","rarity":"Uncommon","chance":11.06}]},{"_id":"ee45ef749a4c8267d8bc187c6cebb8c0","enemyName":"Kela De Thaym","enemyItemDropChance":"25.00","items":[{"_id":"5c4e636c4c8a459c190b972698c0fe13","itemName":"Twin Kohmak Blueprint","rarity":"Common","chance":100}]},{"_id":"024fc70d51074c9f42ee7f917c37b4ed","enemyName":"Juggernaut","enemyItemDropChance":"10.00","items":[{"_id":"00d2e2ced64bb9e06ef7535acee6199e","itemName":"Pherliac Pods Blueprint","rarity":"Common","chance":100}]},{"_id":"d5d0ff699236b032553a1695b104fd77","enemyName":"Hyekka Master","enemyItemDropChance":"1.50","items":[{"_id":"749e658878b5e4d4395cb5a5e88afc81","itemName":"Kavat Incubator Upgrade Segment Blueprint","rarity":"Common","chance":100}]},{"_id":"7687ee02363faf9c6d0605f53e59e550","enemyName":"Juggernaut Behemoth","enemyItemDropChance":"10.00","items":[{"_id":"00d2e2ced64bb9e06ef7535acee6199e","itemName":"Pherliac Pods Blueprint","rarity":"Common","chance":100}]},{"_id":"ee45ef749a4c8267d8bc187c6cebb8c0","enemyName":"Kela De Thaym","enemyItemDropChance":"25.00","items":[{"_id":"5c4e636c4c8a459c190b972698c0fe13","itemName":"Twin Kohmak Blueprint","rarity":"Common","chance":100}]},{"_id":"91b73255556a855d0c86c1903049472e","enemyName":"Grineer Warden","enemyItemDropChance":"1.00","items":[{"_id":"9568cc883b465bc94c14e0d206506b2d","itemName":"Gorgon Blueprint","rarity":"Common","chance":100}]},{"_id":"6ba71181fe67be210250c59fdf06ecb8","enemyName":"Arid Heavy Gunner","enemyItemDropChance":"1.00","items":[{"_id":"9568cc883b465bc94c14e0d206506b2d","itemName":"Gorgon Blueprint","rarity":"Common","chance":100}]},{"_id":"3a4eaf790e4aea8cec2f22ed8d3fc786","enemyName":"Jack O'Naut","enemyItemDropChance":"10.00","items":[{"_id":"00d2e2ced64bb9e06ef7535acee6199e","itemName":"Pherliac Pods Blueprint","rarity":"Common","chance":100}]},{"_id":"e8cc7b1f1e1936f7bc4fe9207d599cd0","enemyName":"Corrupted Warden","enemyItemDropChance":"1.00","items":[{"_id":"9568cc883b465bc94c14e0d206506b2d","itemName":"Gorgon Blueprint","rarity":"Common","chance":100}]},{"_id":"d6516e9a8b30796492a117c68f6fe08f","enemyName":"Kuva Hyekka Master","enemyItemDropChance":"1.50","items":[{"_id":"749e658878b5e4d4395cb5a5e88afc81","itemName":"Kavat Incubator Upgrade Segment Blueprint","rarity":"Common","chance":100}]},{"_id":"4f1c701bacf8ebabfbee83c847725b70","enemyName":"Fissure Corrupted Enemy","enemyItemDropChance":"3.00","items":[{"_id":"c1f5b50e5f850e22df424e2bc4948259","itemName":"Harrow Chassis Blueprint","rarity":"Common","chance":100}]}]}
```

Comment:
non-passive:
- [ ] `blueprintDropChance` should be retained, `enemyItemDropChance` should be additional
- [ ] `mods` should be retained, `items` should be additional
- [ ] `modName` should be retained, `itemName` should be additional